### PR TITLE
Add model parallelism

### PR DESCRIPTION
**Pull request by hyunwoongko**
We add our open sources related with the Transformers :)
The half moon is used to indicate that only a part of the model is running on a specific GPU.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,11 @@ This is a list of some wonderful open-source projects & applications integrated
 * [lightseq](https://github.com/bytedance/lightseq) (from ByteDance) - A high performance inference library for sequence processing and generation implemented in CUDA.
 * [FastSeq](https://github.com/microsoft/fastseq) (from Microsoft) - Efficient implementation of popular sequence models (e.g., Bart, ProphetNet) for text generation, summarization, translation tasks etc.
 
+## 🌗 Model Parallelism
+*Parallelization models across multiple GPUs.*
+* [Parallelformers](https://github.com/tunib-ai/parallelformers) (from TUNiB) - A library for model parallel deployment.
+* [OSLO](https://github.com/tunib-ai/oslo) (from TUNiB) - A library that supports various features to help you train large-scale models.
```
Comment (stas00) [inline README.md:55]:
```suggestion
* [OSLO](https://github.com/tunib-ai/oslo) (from TUNiB) - A library that supports various features to help you train large-scale models.
* [Deepspeed](https://github.com/microsoft/DeepSpeed) (from Microsoft) - Deepspeed-ZeRO - scales any model size with zero to no changes to the model. [Integrated with HF Trainer](https://huggingface.co/docs/transformers/master/main_classes/deepspeed).
* [fairscale](https://github.com/facebookresearch/fairscale) (from Facebook) - Implements ZeRO protocol as well. [Integrated with HF Trainer](https://huggingface.co/docs/transformers/master/main_classes/trainer#fairscale).
```

Let's add ZeRO then as well.

File: README.md [hunk 2]
```diff
@@ -49,6 +49,11 @@ This is a list of some wonderful open-source projects & applications integrated
 * [lightseq](https://github.com/bytedance/lightseq) (from ByteDance) - A high performance inference library for sequence processing and generation implemented in CUDA.
 * [FastSeq](https://github.com/microsoft/fastseq) (from Microsoft) - Efficient implementation of popular sequence models (e.g., Bart, ProphetNet) for text generation, summarization, translation tasks etc.
 
+## 🌗 Model Parallelism
```
Comment (stas00) [inline README.md:52]:
```suggestion
## 🌗 Model Scalability
```

Let's make a generic section on scalability, where parallelism is one of the ways.

**Review by stas00 — CHANGES_REQUESTED** [review]
Thank you for adding these.

Added Deepspeed/fairscale as well.

File: README.md (hunk 2 above)
Comment (hyunwoongko) [reply README.md:52]: Cool !

**Comment by hyunwoongko** [comment]
I commit them !

**Review by stas00 — APPROVED** [review]
Looks good, thank you!

**Pull request merged**